gossip/src/crds_value.rs

```diff
@@ -24,7 +24,7 @@ use {
 #[derive(Serialize, Clone, Debug, PartialEq, Eq)]
 pub struct CrdsValue {
     signature: Signature,
-    data: CrdsData,
+    pub data: CrdsData,
```

Comment:
there is a getter for this. making this field pub is a bad idea since it can break logic for hash.